UnityProject/Assets/Scripts/Items/Others/Restraint.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Restraint : Interactable<HandApply>
+{
+	/// <summary>
+	/// How long it takes to apply the restraints
+	/// </summary>
+	public float applyTime;
+
+	/// <summary>
+	/// How long it takes for another person to remove the restraints
+	/// </summary>
+	public float removeTime;
+
+	// TODO: Add time it takes to resist out of handcuffs
+
+	/// <summary>
+	/// Sound to be played when applying restraints
+	/// </summary>
+	public string sound = "Handcuffs";
+
+	protected override bool WillInteract(HandApply interaction, NetworkSide side)
+	{
+		PlayerMove targetPM = interaction.TargetObject?.GetComponent<PlayerMove>();
```

Comment:
This will handle all the default interact checks (is player in crit state, in range... etc)
```suggestion
		if (!base.WillInteract(interaction, side)) return false;
		
		PlayerMove targetPM = interaction.TargetObject?.GetComponent<PlayerMove>();
```